src/components/DebitCard.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Row, Col, Typography } from 'antd';
+import './DebitCard.css';
+import chip from '../static/chip.png';
+import master from '../static/master.svg';
+
+const { Title, Text } = Typography;
+
+export default function DebitCard(props) {
+    return (
+        <div className="card-container">
+            <Row>
+                <Col>
+                    <Title style={{ color: 'white', fontFamily: 'serif' }}>BAIT BANK</Title>
+                </Col>
+            </Row>
+            <Row>
+                <Col>
+                    <img className="image-logo" alt="chip" src={chip}></img>
+                </Col>
+            </Row>
+            <Row style={{ marginTop: 10 }}>
+                <Col>
+                    <Title level={2} style={{ color: 'white', fontFamily: 'monospace' }}>{props.cardNum}</Title>
```

Comment:
maybe can move this styling into css file

Reply:
done